app/doc/responses.go

```diff
@@ -147,6 +147,27 @@ type updatedGroupRelationsResponse struct {
 	}
 }
 
+// enum: [invalid, success, unchanged, not_found]
+type memberRemovalResult string
+
+// OK. Success response with the per-user deletion statuses
+// swagger:response
+type deletedMembersResponse struct {
```

Comment:
Why not encoding the response fully in the spec comment directly?

Reply:
Fixed